README.md

```diff
@@ -23,10 +23,23 @@ spring:
       enabled: true
       prefixed: true
       paths: /mnt/secrets/this,/mnt/secrets/that,/mnt/secrets/other
+      aliases:
+        this.secret1: SECRET1
+        that.secret1: SECRET2
 ```
 By default both _enabled_ and _prefixed_ are true. This means that properties are
 loaded from the given paths (i.e. enabled) and the file parent directory is used 
 as prefix in the property name. 
+This means that the above configuration is exactly as this:
+```yaml
+spring:
```

Comment:
Something for future and not related to this PR, may be we need to rename it to "some otheR" custom property than spring.cloud

Reply:
Ok, let's agree on a meaningful name for this and then I can make the change in the next PR